test/unit/spec/common/cellComponents/tabs/jobStatusTable-Spec.js

```diff
@@ -452,15 +455,28 @@ define([
                 error,
                 request: error.source,
             },
-            { [jcm.CHANNEL.JOB]: jobId },
+            [jcm.CHANNEL.JOB],
+            jobId,
             jcm.MESSAGE_TYPE.ERROR
         );
     }
 
-    function sendBusMessage(ctx, message, channel, type) {
+    /**
+     * send a jcm.MESSAGE_TYPE.<type> message over the bus
```

Comment:
Set up a fairly standardised way to set up sending job messages over the bus. Since this PR is going to involve editing the structure of job messages, it is much easier to fix if all the message-related stuff in tests is centralised in one place. Thus in the following files, I have consolidated all the bus message sending into a single place and used a very similar function in each file.